File: 10 - IFPB2/app/src/main/java/com/example/ifpb/ifpb/Cadastro.kt
package com.example.ifpb.ifpb

class Cadastro{
    private var lista: ArrayList<Campus>

    constructor(){
        this.lista = ArrayList()
    }

    fun add(campus: Campus){
        this.lista.add(campus)
    }

    fun get(index: Int): Campus{
        return this.lista.get(index)
    }

    fun get(): ArrayList<Campus>{
        return this.lista
    }

    fun update(campus: Campus){

    }

    fun size(): Int{
        return this.lista.size
    }
}

File: iOS/05 - iChoco/iChoco/model/Cadastro.swift
//
//  Cadastro.swift
//  iChoco
//
//  Created by IFPB on 31/01/19.
//  Copyright © 2019 IFPB. All rights reserved.
//

import Foundation

class Cadastro: NSObject {
    var lista: Array<Chocolate>!
    
    override init() {
        self.lista = Array<Chocolate>()
    }
    
    func add(chocolate: Chocolate) {
        self.lista.append(chocolate)
    }
    
    func get() -> Array<Chocolate> {
        return self.lista
    }
    
    func get(index: Int) -> Chocolate {
        return self.lista[index]
    }
    
    func size() -> Int {
        return self.lista.count
    }
    
    func del(index: Int) {
        self.lista.remove(at: index)
    }
}
